vcr/issuer/leia_store.go

```diff
@@ -39,36 +41,35 @@ type leiaIssuerStore struct {
 
 // NewLeiaIssuerStore creates a new instance of leiaIssuerStore which implements the Store interface.
 func NewLeiaIssuerStore(dbPath string) (Store, error) {
-	store, err := leia.NewStore(dbPath, false)
+	store, err := leia.NewStore(dbPath)
 	if err != nil {
 		return nil, fmt.Errorf("failed to create leiaIssuerStore: %w", err)
 	}
-	collection := store.Collection("issuedCredentials")
+	collection := store.JSONCollection("issuedCredentials")
```

Comment:
why don't we treat revocations as JSONLD?

Reply:
This is just the lib update. To JSON-LD happens in the next PR. It has not been decided yet to migrate the other DBs as well. Will be in separate PRs if so